tests/e2e/pom/Base.page.ts

```diff
@@ -0,0 +1,32 @@
+import { Page, Locator, expect} from '@playwright/test'
+
+export class BasePage {
+    readonly page: Page;
+    readonly errorMessage: Locator;
+    readonly successMessage: Locator;
+    readonly pageHeading: Locator;
+    readonly logoutButton: Locator;
+    
+
+    constructor(page: Page) {
+        this.page = page;
+        this.errorMessage = page.locator('#flash_alert');
+        this.successMessage = page.locator('#flash_notice');
+        this.pageHeading = page.locator('h2');
+        this.logoutButton = page.locator('a[href="/logout"]');
+    }
+
+    async goto(path = '/') {
+    await this.page.goto(path);
+}
```

Comment:
keep an eye for indentations, and empty lines, so the file would look neat and tidy.

For immediate results in vscode > settings > search for format > check _Format On Save_
For more formatting functions - Prettier plugin can be used (and configured project wise)